### DISCUSSION TITLE
Let's enable algobot on this repo

### DISCUSSION THREAD

**Discussion by cclauss** [TheAlgorithms/Python#8575] [Team Posts]
@poyea Can you please set up this app https://github.com/dhruvmanila/algorithms-keeper | https://github.com/apps/algorithms-keeper in this repo?  @dhruvmanila has created the app for us to automate the review of PRs based on our standards.

**Comment by dhruvmanila** [comment]
I've sent the request for the installation, someone who has access to the account @TheAlgorithms will have to accept it.

**Comment by poyea** [comment]
I don't have the permission neither so I've submitted a request.

**Comment by dhruvmanila** [comment]
Oh that means only an owner can accept it. We will have to ping them :)

**Comment by dhruvmanila** [comment]
Any luck on contacting the owners?

**Comment by dhruvmanila** [comment]
The bot is live now!

**Comment by cclauss** [comment]
YEAH!!!  Thanks for your persistance!
